config/plugin/windows_defender_problem.ps1

```diff
@@ -0,0 +1,12 @@
+# This plugin checks to see if windows defender detects any threats to the node.
+
+$OK=0
+$NONOK=1
+
+$windowsDefenderThreats = Get-MpThreatDetection
```

Comment:
Updated to use Get-MpThreat because there are fields we can use that can help us determine whether a thread is active or has been executed on the windows node.